core/code/utils_misc.js

```diff
@@ -474,6 +474,42 @@ window.clampLatLngBounds = function (bounds) {
   return L.latLngBounds(clampLatLng(SW), clampLatLng(NE));
 }
 
+
+/*
+pnpoly Copyright (c) 1970-2003, Wm. Randolph Franklin
```

Comment:
Please check this with edge cases, such as "portal on the vertice", "portal on the edge".

Reply:
in fact, pnpoly (by Franklin) doesn't handle well boundaries
it may not be the algorithm we are looking for...
But portals on edge are not common, so just checking vertices may be enough

Reply:
https://github.com/IITC-CE/ingress-intel-total-conversion/issues/552

> so just checking vertices may be enough

ok

Reply:
Library of my choice: point-in-polygon-hao
Size (min): ~700b